### PR TITLE
Improved module/package docstrings

### PR DESCRIPTION
**Pull request by Samuwhale**
This is a first step towards improving the docstrings. Proper docstrings will help the automatically generated API be more usable and informative, it will also help using the package itself.

Next step will be class and function docstrings. But for now, module level docstrings.

I have tried to keep them under 79 characters per line, but in some cases an exception had to be made.

### CODE REVIEW

File: metasyn/var.py [hunk 1]
```diff
@@ -14,15 +14,21 @@
 
 
 class MetaVar():
-    """Meta data variable.
+    """Metadata variable.
 
-    Acts as a base class for specific types of variables, but also as a
-    launching pad for detecting its type.
+    MetaVar is a structure that holds all metadata needed to generate a
```
Comment (qubixes) [inline metasyn/var.py:19]: Would be interesting to rename to MetaSeries at some point?

**Review by qubixes — CHANGES_REQUESTED** [review]
Thanks a lot! I updated the PR  a bit already to remove some linting errors. I really like the new docstrings, more descriptive, without being too verbose! I have added some small comments/suggestions, but nothing major.

File: metasyn/distribution/continuous.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"""Implemented floating point distributions."""
+"""Module implementing uniform (floating point) distributions."""
```
[the remark this answers is not in the document]
Comment (Samuwhale) [reply metasyn/distribution/continuous.py:1]: big whoops

File: metasyn/__init__.py [hunk 3]
```diff
@@ -1,8 +1,20 @@
 """Metasyn: a package for creating synthetic datasets.
 
-One part concerns the creation of the statistical metadata from the
-original data, while the other part creates a synthetic dataset from the
-metadata.
+Metasyn has three main functionalities:
+
+1. Estimation: Metasyn can create a MetaFrame from a dataset.
+A MetaFrame is essentially a fitted model that characterizes the structure of
```
[the remark this answers is not in the document]
Comment (Samuwhale) [reply metasyn/__init__.py:6]: Sounds good to me!

File: metasyn/__init__.py [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+the original dataset without storing actual values.
+It captures individual distributions and features and enables generation of
+synthetic data based on it.
+
+2. Serialization: Metasyn can export a MetaFrame into an easy to read GMF file.
```
[the remark this answers is not in the document]
Comment (Samuwhale) [reply metasyn/__init__.py:11]: Agreed!

File: metasyn/distribution/__init__.py [hunk 5]
```diff
@@ -1,8 +1,8 @@
-"""Distributions for variables.
+"""Package providing different distributions used in metasyn.
 
-These distributions can be fit to datasets/series so that the synthesis is
-somewhat realistic. The concept of distributions here is not only for
-numerical data, but also for generating strings for example.
+Each distribution class provides methods for fitting the distribution to data,
+generating synthetic data, and other functionality specific to the type of
```
[the remark this answers is not in the document]
Comment (Samuwhale) [reply metasyn/distribution/__init__.py:4]: I wrote it thinking that distributions might have specific functionality. I do realise however that that is not the case. I'll rewrite it.

[the remark this answers is not in the document]
Comment (Samuwhale) [reply metasyn/distribution/__init__.py:4]: I've changed it a bit, does this make more sense?

**Comment by Samuwhale** [comment]
I have applied the changes! 

I will also create an issue to update the frontpage of the docs to reflect the content of the metasyn `__init__.py`, and to create an issue to potentially rename MetaVar to MetaSeries.

**Review by qubixes — APPROVED** [review]
Looks good now!

**Pull request merged**